server/planning/assignments/assignments.py

```diff
@@ -153,14 +156,11 @@ async def get_archive_items_for_assignment(self, assignment):
 
         query = {
             "query": {
-                "filtered": {
-                    "filter": {
-                        "bool": {
-                            "must": {"term": {"assignment_id": str(assignment[ID_FIELD])}},
-                        }
-                    }
-                }
-            }
+                "bool": {
+                    "must": {"term": {"assignment_id": str(assignment[ID_FIELD])}},
+                },
+            },
+            "size": get_app_config("ASSIGNMENTS_LINKED_ITEMS_SIZE", 500),
```

Comment:
For future reference, we also have the newer [get_config](https://github.com/superdesk/superdesk-core/blob/develop/superdesk/core/app.py#L53) which is more type safe.